server/webhookNotifier.ts

```diff
@@ -172,6 +170,7 @@ const send = async (models, content: WebhookContent) => {
   await Promise.all(chainOrCommwebhookUrls
     .filter((url) => !!url)
     .map(async (url) => {
+      console.log(url)
```

Comment:
Let's remove console log here.

Reply:
All console logs have been removed, thank you!